src/components/UsersModal/UsersModal.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+
+export const UsersModal = ({ open, handleClose, data }) => {
+  return (
+    <div>
+      <p>UserKa Naam</p>
+      <p>dsdsd</p>
+    </div>
+  );
```

Comment:
I think this is redundant code, you forgot to remove

Reply:
Yeah will do